test/fixtures/label/calloutHorizontalPosition.js

```diff
@@ -58,6 +57,99 @@ module.exports = {
               type: 'point',
               xValue: 'May',
               yValue: 10
+            },
+            text3: {
```

Comment:
Wondering why in this fixture we don't have `point1`? Can you add that to be consistent

Reply:
I'm wondering as well because I have changed yesterday and I didn't recognize it...
I'm pushing it.

Reply:
done